Furball.Engine/Engine/Graphics/Drawables/UiElements/UiTextBoxDrawable.cs

```diff
@@ -17,6 +19,7 @@ public class UiTextBoxDrawable : TextDrawable {
         /// </summary>
         public float TextBoxWidth;
         private bool _selected;
+        private bool _isInContainerDrawable;
```

Comment:
`_isInContainerDrawable` is no longer needed in the current implementation on `master`, it seems that this file is very out of date

Reply:
so do i remove everything surrounding this or?

Reply:
> so do i remove everything surrounding this or?

Id reccomend going through `master` and comparing to this branch for this file, and seeing whats still left over, as i probably missed some stuff